src/backend/marsha/core/management/commands/clean_mediapackages.py

```diff
@@ -0,0 +1,134 @@
+"""Clean orphan mediapackages management command."""
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+import boto3
+
+
+aws_credentials = {
+    "aws_access_key_id": settings.AWS_ACCESS_KEY_ID,
+    "aws_secret_access_key": settings.AWS_SECRET_ACCESS_KEY,
+    "region_name": settings.AWS_S3_REGION_NAME,
+}
+
+# Configure medialive client
+medialive_client = boto3.client("medialive", **aws_credentials)
+
+# Configure mediapackage client
+mediapackage_client = boto3.client("mediapackage", **aws_credentials)
+
+
+def _recursive_call(  # nosec
+    function, key, params=None, next_token=None, next_token_key="NextToken"
+):
+    """
+    Recursive generic call.
+
+    Returns a list of items from the same key in response.
+    """
+    if not params:
+        params = {}
+
+    if next_token:
+        params[next_token_key] = next_token
+
+    response = function(**params)
+    items = response.get(key)
+    next_token = response.get(next_token_key)
+
+    if next_token:
+        items.extend(_recursive_call(function, key, params, next_token, next_token_key))
+    return items
+
+
+def list_mediapackage_channels():
```

Comment:
I think all these helper functions should be defined in the `utils/medialive_utils.py` file